GameData/RealismOverhaul/RO_SuggestedMods/SpaceShuttleSystem/WingSurfaces.cfg

```diff
@@ -19,21 +19,22 @@
 	}
 	
 	!MODULE[ModuleLiftingSurface]{}
+	//new MAC value taken from https://archive.org/details/nasa_techdoc_19810067693
 	%MODULE[FARWingAerodynamicModel]
 	{
-		%MAC = 8.62
+		%MAC = 12.06	//8.62
 		%MidChordSweep = 33.6
-		%b_2 = 8.95
+		%b_2 = 8.95	
 		%TaperRatio = 0.185
-		%rootMidChordOffsetFromOrig = 0, 0, 0
+		%rootMidChordOffsetFromOrig = 0, 0.5, 0
 		%massOverride = 0
 	}
 }
 @PART[WR_SHUTTLE|ShuttleWingR]:FOR[RealismOverhaul]
 {
 	@MODULE[FARWingAerodynamicModel]
 	{
-		%rootMidChordOffsetFromOrig[0] = 6.91
+		%rootMidChordOffsetFromOrig = 6.91, 0.5, 0
```

Comment:
The previous line here changed the first value only. Is there a need for setting the whole vector when only the first value changes?

Reply:
Huh, let's leave it as it is now then!